### DISCUSSION TITLE
pglite in vscode notebooks

### DISCUSSION THREAD

**Discussion by DanielFGray** [electric-sql/pglite#251] [Show and tell]
![Screenshot_20240823_194206](https://github.com/user-attachments/assets/035c91a8-ce55-4a51-a5d8-268aa53b2c89)
I've been working on putting pglite into vscode notebooks
https://postgres-playground.dfg.codes
I still have a lot of ideas to flesh out
it's [here on github](https://github.com/danielfgray/postgres-playground), contributions welcome

**Comment by samwillis** [comment]
This is supper cool @DanielFGray! I would love to share it on Twitter when you feel you are ready for it.

**Reply by DanielFGray** [reply]
Thanks @samwillis, feel free to tweet if you like, I'm on twitter with the same name
Maybe it'll help me get some extra eyes in the code too :sweat_smile:
